### PR TITLE
Fixes  Long search term is not wrapped #2880

### LINKED ISSUE #2880: Long search term is not wrapped
Steps to reproduce:
1. Make a search while typing a long search term without spaces (will be quicker on your device) i.e. https://addons.mozilla.org/en-US/android/search/?q=gheriuiergherighsdifghdjgsdklfghdfkjghsdfjksfghsdfgfd

Expected results:
There are no display or layout issues.

Actual results:
The text is not wrapped and causes a layout issue in the "0 results for search_term" header.

Notes/Issues:
- reproducing on new desktop pages also;

Verified on AMO-prod FF54(Win 7, Android 7.2.1).
Screenshots for this issue:
device
desktop

### PR DESCRIPTION
**Pull request by tsl143**
Fixes #2880 

**Before:**
screen shot 2017-08-02 at 11 58 40 am

**After**
screen shot 2017-08-02 at 11 55 23 am

### CODE REVIEW

**Comment by muffinresearch** [comment]
I think this needs a similar rule for the "No results were found for..." part

**Comment by tsl143** [comment]
**Before : message **
screen shot 2017-08-02 at 9 18 52 pm


**After : message**
screen shot 2017-08-02 at 9 19 29 pm

File: src/amo/components/Search/styles.scss [hunk 1]
```diff
@@ -53,4 +53,7 @@
       grid-row: 4;
     }
   }
+  .SearchResults-message {
```
Comment (muffinresearch) [inline src/amo/components/Search/styles.scss]: Needs a newline above.

**Comment by tsl143** [comment]
@muffinresearch done :)

**Comment by muffinresearch** [comment]
> @muffinresearch done :)
But this newline issue was not addressed in lint test, shall we write a lint rule for that?

Yep good call -  I agree it would be better handled by stylelint. If you'd like to file a bug then we can look and see if stylelint has an existing rule for that.

**Pull request merged**